mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/documentdialog/DocumentDialogFragment.java

```diff
@@ -176,9 +182,24 @@ public void validateInput() throws RequiredFieldException {
 
         //Start Uploading Document
         if (documentAction == getResources().getString(R.string.update_document)) {
```

Comment:
@therajanmaurya What did I just see here ? 😱😱 

You are comparing String with ```==``` and not ```equals()```

Please fix this ASAP

Reply:
Sorry, my bad, I did this in starting of GSoC last year, Now we are using `enum` for these type scenarios.

Reply:
I've fixed it.